float32[3] current_position
float32[3] target_position

float32 Euclidean_error
